### PR TITLE
Remove old webinar/events; clean up listings

### PR DESCRIPTION
**Pull request by zchase**
This PR removes old webinars from being listed until we have a recording. It also cleans up some of the video listings on the /superpowers page by removing dates from talks that are videos now.

### CODE REVIEW

**Comment by isaac-pulumi** [comment]
Please embed the video here: https://youtu.be/xI9uWFiEzAM

File: content/webinars/gitops-with-gitlab-and-pulumi-2020-05-20/index.md [hunk 1]
```diff
@@ -71,10 +72,6 @@ main:
 
 # The right hand side form section.
 form:
-    # GoToWebinar webinar key. This key allows us to register people for webinars via the
-    # HubSpot form.
-    gotowebinar_key: "1017103637425347085"
-
     # HubSpot form id.
-    hubspot_form_id: "ebc6064d-756c-4d04-ac63-31ad16780ab5"
+    hubspot_form_id: "87cc0632-52f1-444a-b41c-6775470f4ece"
```
Comment (cnunciato) [inline content/webinars/gitops-with-gitlab-and-pulumi-2020-05-20/index.md:76]: Does this need to be changed for anything in this PR?

**Comment by cnunciato** [comment]
Also, thoughts on my comment about showing dates? I do feel like we should be showing these (at least on the webinar pages themselves).

File: content/webinars/gitops-with-gitlab-and-pulumi-2020-05-20/index.md (hunk 1 above)
Comment (zchase) [reply content/webinars/gitops-with-gitlab-and-pulumi-2020-05-20/index.md:76]: It doesn't need to exist. Will remove.

**Comment by zchase** [comment]
@cnunciato I'll put them back on the webinar pages themself. I do think that `/superpowers` should be as timeless as possible.

**Comment by cnunciato** [comment]
> I do think that `/superpowers` should be as timeless as possible.

Yep, that makes sense to me too. As long as the dates are on the webinar pages themselves, I'm happy. 😄

**Pull request merged**